src/OpenTelemetry.Extensions.Hosting/README.md

```diff
@@ -9,92 +9,66 @@
 dotnet add package --prerelease OpenTelemetry.Extensions.Hosting
 ```
 
-## Usage
-
-### Tracing
-
-#### Simple Configuration
-
-The following example registers tracing using the `ZipkinExporter` and binds
-options to the "Zipkin" configuration section:
-
-```csharp
-services.AddOpenTelemetryTracing((builder) => builder
-    .AddAspNetCoreInstrumentation()
-    .AddHttpClientInstrumentation()
-    .AddZipkinExporter());
+## Overview
+
+The OpenTelemetry.Extensions.Hosting package provides extension methods for
+automatically starting (and stopping) OpenTelemetry tracing (`TracerProvider`)
+and metrics (`MeterProvider`) in [ASP.NET
+ Core](https://learn.microsoft.com/aspnet/core/fundamentals/host/web-host) and
+ [.NET Generic](https://learn.microsoft.com/dotnet/core/extensions/generic-host)
+ hosts. These are completely optional extensions meant to simplify the
+ management of the OpenTelemetry SDK lifecycle.
+
+## Extension method reference
+
+**Note:** The below extension methods target
+`Microsoft.Extensions.DependencyInjection.IServiceCollection`.
+
+**Note:** The below extension methods should be called by application host code
+only. Library authors see: [Registration extension method guidance for library
+authors](../../docs/trace/extending-the-sdk/README.md#registration-extension-method-guidance-for-library-authors).
+
+**Note:** Multiple calls to the below extensions will **NOT** result in multiple
```

Comment:
Also good to highlight that all configurations gets applied to the single provider that gets build